src/CMakeLists.txt

```
@@ -542,6 +543,7 @@ if(BUILD_CLIENT)
 		${GMP_LIBRARY}
 		${JSON_LIBRARY}
 		${LUA_BIT_LIBRARY}
+		${SHA256_LIBRARY}
```

```suggestion
		sha256
```
and the same for the server. `sha256` is a *target,* not just library name so there is no point in using a variable.

---

I had not done this because I was not sure if it's possible in old cmake (but it is, according to doc), and because of consistency.
Changed it.